mirdata/datasets/tiny_4mula.py

```diff
@@ -0,0 +1,319 @@
+            self._data_home, self._track_paths["melspectrogram"][0]
+        )
+
+    @property
+    def music_id(self):
+        return self._track_metadata.get("music_id")[0]
+
+    @property
+    def music_name(self):
+        return self._track_metadata.get("music_name")[0]
+
+    @property
+    def music_lang(self):
+        return self._track_metadata.get("music_lang")[0]
+
+    @property
+    def music_lyrics(self):
+        return self._track_metadata.get("music_lyrics")[0]
+
+    @property
+    def art_id(self):
+        return self._track_metadata.get("art_id")[0]
+
+    @property
+    def art_name(self):
+        return self._track_metadata.get("art_name")[0]
+
+    @property
+    def art_rank(self):
+        return self._track_metadata.get("art_rank")[0]
+
+    @property
+    def main_genre(self):
+        return self._track_metadata.get("main_genre")[0]
+
+    @property
+    def related_art(self):
+        return eval(self._track_metadata.get("related_art")[0])
+
+    @property
+    def related_genre(self):
+        return eval(self._track_metadata.get("related_genre")[0])
+
+    @property
+    def related_music(self):
+        return eval(self._track_metadata.get("related_music")[0])
+
+    @property
+    def musicnn_tags(self):
+        return eval(self._track_metadata.get("musicnn_tags")[0])
+
+    @property
+    def _track_metadata(self):
+        metadata = read_csv(self.annotation_path, sep="\t")
+        if not metadata.empty:
+            return metadata
+        return None
+
+    @property
+    def melspectrogram(self):
+        return load(self.melspectrogram_path, allow_pickle=True)
+
+    def to_jams(self):
+        """Get the track's data in jams format
+
+        Returns:
+            jams.JAMS: the track's data in jams format
+
+        """
+        return jams_utils.jams_converter(
+            spectrogram_path=self.melspectrogram_path,
+            metadata={
+                "art_id": self.art_id,
+                "art_name": self.art_name,
+                "art_rank": self.art_rank,
+                "main_genre": self.main_genre,
+                "music_id": self.music_id,
+                "music_lang": self.music_lang,
+                "music_lyrics": self.music_lyrics,
+                "music_name": self.music_name,
+                "musicnn_tags": self.musicnn_tags,
+                "related_art": self.related_art,
+                "related_genre": self.related_genre,
+                "related_music": self.related_music,
+                "duration": 30,  # approximation
+            },
+        )
+
+
+def load_melspectrogram(fhandle: str) -> ndarray:
+    """Load a tiny_4mula dataset melspectrogram.
+
+    Args:
+        fhandle (str): path pointing to the dataset file
+
+    Returns:
+        numpy.ndarray: melspectrogram
+
+    """
+    print(str(fhandle))
+    if not os.path.isfile(str(fhandle)):
+        raise FileNotFoundError("Dataset not found. Did you run .download()?")
```

Comment:
we are not including this error catch since you would get `FileNotFoundError` anyway if you try to access those

Reply:
Yes... however, returning an objective error message to the user will help in the user experience.